### ISSUE TITLE
Request ability to enable via API and subsequently via Homeassistant

### ISSUE THREAD

**Issue by thowiegit** [SonnenladenGmbH/APsystems-EZ1-API#9]
Es wäre schön wenn man Bluetooth über die lokale API bei Bedarf wieder einschalten kann.

Aktuell muss man kurzzeitig den Strom zum Inverter trennen oder sein Wifi vorübergehend abschalten um Bluetooth bei Bedarf wieder zu aktivieren, das ist doch eher umständlich.

Es wäre daher schön dies mit über die lokale API und aus Homeassistant heraus erledigen zu können.

Denn schließlich wird man Bluetooth nicht dauerhaft aktiv lassen wollen um dritten keine Spielwiese offen zu lassen.

**Comment by DirkBaumeister** [comment]
Just adding my comment here also to support this request :) Please let us know if there is any progress :)

**Comment by Schw3n** [comment]
Dear all, 
I would like to take this opportunity to briefly describe my experiences. I was very pleased to convince my neighbor to buy a plug-in solar device.
Of course, he also bought an AP System EZ1. As luck would have it, I've just received a text message from him saying “hey, I've just dialed into your inverter via bluetooth.”
If my neighbor can do it, what are strangers capable of?

I would therefore like to support this request.

**Comment by amjadsaadeh** [comment]
Yes having the ability to turn off bluetooth is quite important. But it looks like that the API provided by EZSystems does not support this, at least couldn't find any docs about this.

**Comment by lukas-fichtner** [comment]
Any known updates from the vendor about this feature?

**Comment by Schw3n** [comment]
Are there any updates?

**Comment by mawoka-myblock** [comment]
No, not really. I wrote them on June 6th and received a response on June 10th, I quote:

> For your other requests, I will forward them to the R&D team, they will evaluate.


Well... They don't seem to make a lot of that...

And here's the relevant part of my request to them:

> Alright, that's about it for now. If you're interested, there are a few feature requests the users brought to my attention, so make of them what you will:
>
> 1. Authentication for the API (eg. an API key created when activating the local API), or the serial number
> 2. More detailed errors instead of a “FAILED” response out of nowhere
>3. A per-solar-input max power limit and voltage-per-input measurement
>4. Reporting both to the cloud and having the local API available
>5. Direct bluetooth setup from Home Assistant without the EasyPower app
>6. Control Bluetooth using the API
